classes/abstracts/ActionScheduler_Abstract_ListTable.php

```diff
@@ -574,7 +655,7 @@ protected function display_header() {
 	 */
 	protected function display_admin_notices() {
 		foreach ( $this->admin_notices as $notice ) {
-			echo '<div id="message" class="' . $notice['class'] . '">';
+			echo '<div id="message" class="' . esc_html( $notice['class'] ) . '">';
```

Comment:
`esc_attr()` could be a better choice here, _or_ `sanitize_html_class()` (but the latter would only be safe if we are certain multiple classes are not being passed in here):

```suggestion
			echo '<div id="message" class="' . esc_attr( $notice['class'] ) . '">';
```

Reply:
Suggestion added :)